Move some commas around (#71)
soupi/learn-haskell-blog-generator: src/05-glue/03-project.md

### Package metadata

> The first part should be fairly straightforward from the comments, maybe, except for:

Reviewer: ```suggestion
The first part should be fairly straightforward from the comments, maybe except for:
```

  -- other-modules:
```

Also, note that we've added an additional *hierarchy* for our modules and defined

> a different source directory. This means we must move the files around

Reviewer: ```suggestion
a different source directory (`src`). This means we will need to move the files around
```

[stack.yaml](https://docs.haskellstack.org/en/stable/yaml_configuration/#project-specific-config)
files are used by `cabal` and `stack` respectively to add additional information on *how
to build the package*. While `cabal.project` isn't necessary to use `cabal`, `stack.yaml`

> is necessary to use `stack`, so we will cover it briefly.

Reviewer: ```suggestion
is necessary in order to use `stack`, so we will cover it briefly.
```

and use the GHC on PATH to build the project.

Cabal caches packages between projects, so if a new project uses the same packages
with the same versions (and the same flag settings), they will not need to be reinstalled.

> In older versions of cabal, packages could be installed globally or in sandboxes.
> In each sandbox (and globally), there could only be one version of a package installed,
> and users would usually create different sandboxes for different projects without caching
> packages between projects.
>
> With the new build system implementation, multiple versions of the same package can be
> installed globally, and for each project, cabal will (try to) choose a specific version for each
> package dependency such that they all work together without needing sandboxing.
> This change helps us increase sharing of built packages while avoiding conflicts and manual
> handling of sandboxes.

> > Note: The new build system implementation is now the default, and Cabal commands do not need
> > to be prefixed with `v2-`, but the Cabal documentation will still mention the prefix to
> > refer to the new commands.

Reviewer: No longer relevant.

```suggestion
```
